packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -195,6 +209,56 @@ const getAppealType = (appealCaseData) => {
 	return `${appealCaseData.appealTypeName} appeal`;
 };
 
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData
+ * @returns {boolean}
+ */
+const isEligibilityCompleted = (appealCaseData) => {
+	if (appealCaseData?.appeal?.state === APPEAL_STATE.DRAFT) {
+		const eligibility = appealCaseData.appeal.eligibility;
+		return (
+			eligibility !== undefined &&
+			eligibility.enforcementNotice !== null &&
+			eligibility.applicationDecision !== null &&
+			eligibility.applicationCategories !== null
+		);
+	}
+	return true;
+};
+
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData
+ * @returns {boolean}
+ */
+const hasDecisionDate = (appealCaseData) => {
+	if (appealCaseData?.appeal?.state === APPEAL_STATE.DRAFT) {
+		return appealCaseData.appeal.decisionDate !== null;
+	}
```

Comment:
I don't think this is required, `AppealCaseWithAppellant` can't be a draft.

Reply:
Amended function to use `isAppealSubmission`